app/src/ui/publish-repository/publish.tsx

```diff
@@ -72,23 +93,35 @@ export class Publish extends React.Component<IPublishProps, IPublishState> {
       startingTab = PublishTab.Enterprise
     }
 
-    const publishSettings = {
+    const GHEPublicationSettings = {
       name: props.repository.name,
       description: '',
       private: true,
-      org: null,
+    }
+
+    const dotcomTabState: IDotcomTabState = {
+      kind: 'dotcom',
+      settings: { ...GHEPublicationSettings, kind: 'dotcom', org: null },
```

Comment:
I would use `PublicationSettings` instead of `GHEPublicationSettings`.

`dotcomTabState` using `GHEPublicationSettings` seems confusing to me.

Reply:
That's because it's a mistake!!!

Good catch! 😆